oneview/resource_server_profile_template.go

```diff
@@ -1455,17 +1497,25 @@ func resourceServerProfileTemplateCreate(d *schema.ResourceData, meta interface{
 								}
 							}
 						}
-						bootOptionV3 := ov.BootOptionV3{
-							BootVlanId: bootItem["boot_vlan_id"].(int),
-						}
+
 						bootOptions = ov.BootOption{
 							Priority:         bootItem["priority"].(string),
-							BootOptionV3:     bootOptionV3,
 							EthernetBootType: bootItem["ethernet_boot_type"].(string),
 							BootVolumeSource: bootItem["boot_volume_source"].(string),
 							Iscsi:            &iscsi,
 							Targets:          bootTargets,
 						}
+						if bootItem["boot_vlan_id"].(string) != "" {
+							val, err := strconv.Atoi(bootItem["boot_vlan_id"].(string))
```

Comment:
Not sure if this specific implementation required

Reply:
As discussed its done to bypass the drifts.